ci-operator/step-registry/single-node/conf/aws/graceful-shutdown/single-node-conf-aws-graceful-shutdown-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Reserve more system memory per node than a typical multi-node cluster
+# to facilitate E2E tests on a single node.
+cat > "${SHARED_DIR}/manifest_single-node-reserve-sys-mem-kubeletconfig.yml" << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: single-node-reserve-sys-mem
+spec:
+  containers:
+    machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/master: ""
+  kubeletConfig:
+    shutdownGracePeriod: 600s
+    shutdownGracePeriodCriticalPods: 300s
+    systemReserved:
```

Comment:
Why is the graceful shutdown file defining system reserved memory? It is already defined by the config-e2e step